payments_config/__init__.py

```diff
@@ -45,6 +46,7 @@ def __init__(self, seller, config):
         self.currency = 'USD'
         self.seller = seller
         self.recurrence = None
+        self.user_identification = None
```

Comment:
I think it's dangerous to define this by default because if someone mistakenly leaves off `user_identification` then it would be wrong to assume they meant "no authentication required." This validator should raise a ValueError if nothing is defined at all which means the configuration for one-time donations should say explicitly `'user_identification': None`.